README.md

```diff
@@ -36,3 +42,49 @@ SonarJS rules for ESLint
 [`no-use-of-empty-return-value`]: ./docs/rules/no-use-of-empty-return-value.md
 [`prefer-immediate-return`]: ./docs/rules/prefer-immediate-return.md
 [`prefer-single-boolean-return`]: ./docs/rules/prefer-single-boolean-return.md
+
+## Prerequisites
+
+Node.js (>=6.x).
```

Comment:
@stas-vilchik we don't have any requirements to eslint version, do we?

Reply:
we do, but they're not so important IMO

```json
"peerDependencies": {
  "eslint": "^3.0.0 || ^4.0.0"
},
```